using-timescaledb/distributed-hypertables.md

```diff
@@ -103,6 +103,90 @@ the data node's data is outside the retention window,
 [`drop_chunks`][drop_chunks] can be used to delete the data on the data
 node.
 
+### Replication Factor for Distributed Hypertables [](replication-factor)
+
+A hypertable can have a *replication factor* that tells how many
+replicas of chunks that should be created. The default replication
+factor is 1, which means that each row of data is written to a single
+chunk. If you set the replication factor of a hypertable to a higher
+value, chunks will be replicated and each row of data will be written
+to that many chunks.
+
+With a replication factor of one, data is written to the data nodes
+using a one-phase commit, which basically means that every transaction
+on the access node opens a similar transaction on the data
+node. Committing the transaction involves sending a commit message to
+the data node and no additional coordination.
+
+With a higher replication factor, data will be written to the chunks
+using two-phase commit. This protocol is more complicated, but it
+guarantees that the chunks are kept in sync on all data nodes and that
+failure to write to one of the data nodes will generate an error.
+
+To create a hypertable with a replication factor higher than 1, you
+can use the `replication_factor` parameter to
+[`create_distributed_hypertable`][create_distributed_hypertable]. For
+example, if you want to create a hypertable where each chunk has three
+replicas:
+
+```sql
+SELECT create_distributed_hypertable('conditions', 'time', 'location',
+	replication_factor => 3);
+```
+
+If you already have a hypertable and instead want to change the
+replication factor, you can use the function
+[`set_replication_factor`][set_replication_factor].
+
+```sql
+SELECT set_replication_factor('conditions', 3);
+```
+
+Having a replication factor larger than one is a simple way to ensure
+that you have redundancy of the data and that a crash of a data node
+will not cause you to lose the data. This, however, comes at the cost
+of an increased average commit time and more data to manage.
+
+Transactions that write to the distributed table will be slower on
+average with multiple replicas (when compared with a single replica)
+for two reasons:
+- The two-phase commit procedure induces additional processing
+  compared to one-phase commit. This is, however, minor and does not
+  significantly affect the commit time.
+- Since there are now more transactions to be committed, the commit
+  time of the two-phase commit (also known as the *global commit*) is
+  the maximum time of the commits on the data nodes (also known as
+  *local commits*), which on average is going to be higher than the
+  average of the commit of each transaction on the data nodes. (The
+  proof of this is left as an excercies for the reader.)
+
+When executing a two-phase commit, the commit will block until it
```

Comment:
I think saying it will "block" is misleading; in particular, the client can be notified of success as soon as the coordinator has received "agree to commit" from all nodes.

In any case, I don't think we should go into this amount of detail about the two-phase commit protocol in our documentation. There are better resources online to point to.

Reply:
I think it is useful to give a brief explanation what causes the delay and also leads into the limitations below (that the data node needs to be replaced). The intention is not to go into details, but give a superficial understanding. Reworded.